src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.jsx

```diff
@@ -90,11 +106,18 @@ const StepIndicator = ({
   return (
     <li
```

Comment:
Maybe change this `li` into a `button`?
When we add this onClick here, it may represent the element better

Reply:
But I did add a role of button to the element, and the step is an item in a list so I think it makes sense no?